app/assets/stylesheets/modules/site-nav.css.scss

```diff
@@ -1,41 +1,100 @@
-.nav {
-  margin: 0;
-  min-width: 100%;
-  max-width: 100%;
-  height: $nav-height;
-  background-color: white;
+$breakpoint: 780px;
```

Comment:
I prefer to keep variables that are used only in one file at the top of that file, but that's a matter of preference. if you have strong feelings about this lmk